docs/src/installation.md

```diff
@@ -131,6 +131,12 @@ pkg install rust-coreutils
 
 ## Windows
 
+### Winget
```

Comment:
I suggest we add `winget` after `conda` in the first line of this file, making cspell happy.

```
<!-- spell-checker:ignore pacman pamac nixpkgs openmandriva conda winget -->
```